packages/commonwealth/client/scripts/views/components/component_kit/cw_text_area.tsx

```diff
@@ -156,6 +162,11 @@ export const CWTextArea = (props: TextAreaProps) => {
           }
         />
       )}
+      {charCount && (
+        <div className="character-count">
+          <CWText type="caption">Character count: {characterCount}/250</CWText>
```

Comment:
250 is in this particular case but in the other situation, the value might be different. Maybe `charCount` could be number instead of boolean and it will be used as a max count

Reply:
you dont need to do input validation changeable, as validation is specific for given component/view. Basic information form in this case. Just make `charCount` dynamic

Reply:
@masvelio 
![79290cfe-6a60-40b9-84f3-dcff7ef3b4d5](https://github.com/user-attachments/assets/ea603b77-5869-493e-986f-2d3ed72ffe67)